src/events/_event-post-macros.html

```diff
@@ -354,13 +371,13 @@ <h2 class="event-venue_heading">{{ city }}</h2>
                   {% endif %}
                 </td>
                 <td class='{{ options.speaker_col_classes }}' data-label="Speakers">
-                    {% for speaker in block.speakers %}
-                        {%- if speaker.url -%}
-                            <a href="{{ speaker.url }}">{{ speaker.name }}, {{ speaker.position }}</a>{{ ';' if loop.index < loop.length }}
-                        {% else %}
-                            {{ speaker.name }}, {{ speaker.position }}{{ ';' if loop.index < loop.length }}
-                        {%- endif -%}
-                    {% endfor %}
+                {% for speaker in block.speakers %}
+                    {%- if speaker.url -%}
+                        <a href="{{ speaker.url }}">{{ speaker.name }}, {{ speaker.position }}</a>{{ ';' if loop.index < loop.length }}
```

Comment:
If I understand it correctly, the desire is to have the semi-colon not appear at the end of list of links. If that's true, then this will happen at the end of the loop since the loop.index starts at 1.